core/utils.ts

```diff
@@ -9,20 +7,30 @@ import { IActionProto, Session } from "df/core/session";
 import { Table } from "df/core/table";
 import { dataform } from "df/protos/ts";
 
+const pathSeperator = (() => {
+  if (typeof process !== "undefined") {
+    return process.platform === "win32" ? "\\" : "/";
+  }
+  return "/";
+})();
+
```

Comment:
```suggestion
const PATH_SEPARATOR = process?.platform === "win32" ? "\\" : "/";
```

Reply:
A very reasonable suggestion! And what I first tried, however, it does not work:

![image](https://user-images.githubusercontent.com/912458/130073479-c08f68bb-6f85-40cb-b43f-6f082b5f1fa2.png)